tests/content-types.js

```diff
@@ -678,75 +692,23 @@ test('Content Type config check attribute type is string', t => {
   t.is(check, 'Attribute type must be a string', 'Attribute type must be a string');
 });
 
-test('Content Type config check attribute as key', t => {
-  const type = {
-    name: 'test',
-    id: 'test',
-    attributes: [{
-      name: 'foo',
-      id: 'foo',
-      type: 'text',
-      key: 'true',
-    }, {
-      name: 'bar',
-      id: 'bar',
-      type: 'text',
-    }],
-  };
-  let check = types.raw.check(type);
-  t.is(check, 'Attribute key must be a boolean', 'Attribute key must be a boolean');
 
-  type.attributes = [{
-    name: 'foo',
-    id: 'foo',
-    type: 'text',
-    key: true,
-  }, {
-    name: 'bar',
-    id: 'bar',
-    type: 'text',
-    key: 'true',
-  }];
-  check = types.raw.check(type);
-  t.is(check, 'Attribute key must be a boolean', 'Attribute type must be a string');
-});
-
-test('Content Type config only one attribute as key', t => {
-  const type = {
-    name: 'test',
-    id: 'test',
-    attributes: [{
-      name: 'foo',
-      id: 'foo',
-      type: 'text',
-      key: true,
-    }, {
-      name: 'bar',
-      id: 'bar',
-      type: 'text',
-      key: true,
-    }],
-  };
-  const check = types.raw.check(type);
-  t.is(check, 'Only one attribute key is allowed', 'Attribute key must be a boolean');
-});
 
 test('configuration check - winner', t => {
   const type = {
     name: 'Foo',
     id: 'foo',
+    identifier: true,
     attributes: [
       {
         type: 'text',
         id: 'text',
```

Comment:
Do we actually have a check in the content model ingestion to ensure that each attribute has an `id` and `type`? `name` isn't actually needed (it can be inherited from the input plugin) but the other two are.